apiserver/facades/client/controller/controller_integration_test.go

```diff
@@ -39,25 +39,6 @@ func (s *ControllerIntegrationSuite) TestWatchModelSummaries(c *gc.C) {
 	watcher, err := s.client.WatchModelSummaries(context.Background())
 	c.Assert(watcher, gc.IsNil)
 	c.Assert(err, gc.NotNil)
-	//c.Assert(err, jc.ErrorIsNil)
```

Comment:
Can I suggest that instead of commenting these out because we aren't cleaning up the resources properly that we just use `c.Skip`.

Some thing like `c.Skip("TODO (alvin) - reimplement when facade moved off of mongo")`

Reply:
Hi tom, im actually removing the comments, should i bring the comments back?

Reply:
We don't want a failing test. I would remove all of the comments and use the `c.Skip`

Reply:
Sure thanks tom!